src/alpine/3.22/helix/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM library/alpine:3.22 AS venv
```

Comment:
Oops, I didn't import this one. I'll do that now.

Reply:
Which one did you import?

Reply:
This should be done now.

Reply:
Ah. Got it.